Built/Borrowed a very simple intro layout

binarycleric opened the pull request:
Borrowed a bootstrap layout (http://startbootstrap.com/templates/stylish-portfolio/index.html) to get a very basic splash page started. Not intended to be a final product but instead it's just something for us to build upon.

binarycleric committed:
Built/Borrowed a very simple intro layout

Borrowed a bootstrap layout
(http://startbootstrap.com/templates/stylish-portfolio/index.html) to get a very
basic splash page started. Not intended to be a final product but instead it's
just something for us to build upon.

Valarissa commented on app/views/visits/index.html.erb:
```diff
@@ -1,4 +1,21 @@
+    <div class="intro-text"> 
+      <h3>
+        Transgress is a content-filtering bypass system developed specifically 
+        to allow access to trans-related resources on public networks 
```
The site is specifically bare bones and doesn't include much text to prevent immediate filtering of the site by the same content blockers we're looking to circumvent. I realize a mission statement is usually a good idea, but I am actively paranoid of /trans(-?)/ being flagged.

binarycleric replied:
:thumbsup: Understandable. Want me to rip that out?

binarycleric replied:
I suppose by that logic I should remove the github link as well.

Valarissa replied:
The github link should be fine because of the fact that transgress is a word without trans-related connotations. I don't think a service would block based on a word that has useful academic meaning.

binarycleric committed:
Removed transgress header text.

Trying to give content filters as little information as possible to avoid
transgress nodes getting blocked.

binarycleric commented:
@Valarissa update pushed. When I have more time I'd be happy to tweak the layout some more. Maybe we can do a pair session in the future?

Valarissa commented:
Thanks so much for this! And yes, pairing would be great!
